### PR TITLE
FXVPN-156: Issues with extended popup view

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
https://github.com/user-attachments/assets/9e9feff2-322f-4c3f-8897-0f7c888f9790

Here we:
- Fix the UI overflow and scrolling
- Consistently show the reset button and enable/disable it as the checkbox is clicked or as a location is set for the site.

### CODE REVIEW

File: src/ui/browserAction/popupPage.js [hunk 1]
```diff
@@ -51,30 +52,36 @@ export class BrowserActionPopup extends LitElement {
...
 
   constructor() {
     super();
     this.pageURL = null;
     this._siteContext = null;
-    browser.tabs.onUpdated.addListener(() => this.updatePage());
-    browser.tabs.onActivated.addListener(() => this.updatePage());
-    vpnController.state.subscribe((s) => (this.vpnState = s));
+    vpnController.state.subscribe((s) => {
+      this.vpnState = null;
```
Comment (strseb) [inline src/ui/browserAction/popupPage.js:63]: Why nulling it?

File: src/ui/browserAction/popupPage.js [hunk 2]
```diff
@@ -51,30 +52,36 @@ export class BrowserActionPopup extends LitElement {
...
+      /** @type {VPNState} */
+      this.vpnState = s;
+    });
     vpnController.servers.subscribe((s) => (this.servers = s));
+    proxyHandler.siteContexts.subscribe((s) => {
+      this._siteContexts = s;
+      console.log(this._siteContexts);
```
Comment (strseb) [inline src/ui/browserAction/popupPage.js:70]: Not sure if that is needed :)

Comment (lesleyjanenorton) [reply src/ui/browserAction/popupPage.js:70]: ha! no we probably don't want to do this.

File: src/ui/browserAction/popupPage.js (hunk 1 above)
Comment (lesleyjanenorton) [reply src/ui/browserAction/popupPage.js:63]: we were nulling it in `updatePage()` but maybe we don't actually need to do this any longer. investigating.

Comment (lesleyjanenorton) [reply src/ui/browserAction/popupPage.js:63]: Removing...

**Pull request merged**